Database/TestDatabaseCache.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+/*
+ * This file is part of the Liip/FunctionalTestBundle
+ *
+ * (c) Lukas Kahwe Smith <smith@pooteeweet.org>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Liip\FunctionalTestBundle\Database;
+
+use Symfony\Component\DependencyInjection\Container;
+use Doctrine\Common\DataFixtures\Executor\AbstractExecutor;
+
+class TestDatabaseCache
+{
+    private $container;
+
+    public function __construct(Container $container)
+    {
+        $this->container = $container;
+    }
+
+    public function getCachedExecutor(TestDatabasePreparator $dbPreparator, array $classNames)
```

Comment:
btw, this method seems to be called only inside this class, so why is it public ?